conjur/user/user_controller.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+"""
+UserController module
+
+This module is the controller that facilitates all user actions
+"""
+
+# Builtins
+import getpass
+import logging
+import sys
+import requests
+
+class UserController():
+    """
+    UserController
+
+    This class represents the Presentation Layer for the User command.
+    """
+    def __init__(self, user_logic, user_resource_data):
+        self.user_logic=user_logic
+        self.user_resource_data=user_resource_data
+
+    def rotate_api_key(self):
+        """
+        Method that calls to rotate api key logic to perform the rotation
+        """
+        resource_to_update, new_api_key = self.user_logic.rotate_api_key(self.user_resource_data.id)
+        sys.stdout.write(f"API key for '{resource_to_update}' was successfully rotated. " \
+                         f"New API key is: {new_api_key}\n")
+
+    def change_password(self):
+        """
+        Method that calls to change password to perform the change
+        """
+        self.prompt_for_password()
+        try:
+            self.user_logic.change_password(self.user_resource_data.new_password)
+        except requests.exceptions.HTTPError as http_error:
+            raise requests.exceptions.HTTPError("Invalid password. " \
```

Comment:
Create an object for it  `PasswordNotComplexEnough`

Reply:
a Conjur CLI-specific exception as in?

Reply:
adding